transit_odp/disruptions/models.py

```diff
@@ -23,3 +23,124 @@ def __repr__(self):
             f"last_updated={self.last_updated!r}, data={self.data.name!r}, "
             f"data_format={self.data_format!r})"
         )
+
+
+class DisruptionReasonsEnum(models.TextChoices):
+    accident = "accident", "Accident"
```

Comment:
is it possible to use pythonic style rather than camel case

Reply:
I've removed this code and its usages as it was not being used anymore